hutch_python/pt_app_config.py

```diff
@@ -0,0 +1,21 @@
+"""
+This file defines an ipython extension to configure the terminal app.
+Currently this configures:
+- disabling the Ctrl+\\ keybind
+"""
+
+import logging
+
+from prompt_toolkit.keys import Keys
+
+from .utils import safe_load
+
+logger = logging.getLogger(__name__)
+
+
+def load_ipython_extension(ipython):
+    # Unbind Ctrl+\\
+    with safe_load('disable ctrl+\\, auto suggestions'):
```

Comment:
Auto suggestions are still showing up for me (which is fine for me personally as I think they're useful...):

<img width="212" alt="image" src="https://user-images.githubusercontent.com/5139267/164336688-05744ad9-7c5f-4308-a3a0-41e13d72060d.png">

Reply:
I took that part out of this PR, since there was some uncertainty as to whether or not we wanted it.  

Previously, I think a few scientists wanted it gone because there was uncertainty around how to ignore the suggestions.  I imagine in time they'll get used to it, but I decided to punt that decision

Reply:
```suggestion
    with safe_load('disable ctrl+\\'):
```